packages/dap/src/report.ts

```diff
@@ -53,14 +53,14 @@ export class PlaintextInputShare implements Encodable {
 
 export class InputShareAad implements Encodable {
   constructor(
-    public taskId: TaskId,
```

Comment:
Self review so I don't forget: This change should revert. It helps to clarify what kind of id it is here